docs/source/quasi_dags.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<p>For a next iteration we can show the problem with Bambi or PyMC with the do operator (maybe create a follow up issue?)</p><p> </p><p>Also, it would be nice to give a concrete example for readers that see this for the same time.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-labs/CausalPy/pull/321/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I was thinking exactly this. I've created issue #327 to keep track of this.</p><p> </p>

Reply:
Great!